File: lib/src/main/java/com/qwwuyu/lib/gson/Exclusion.java
package com.qwwuyu.lib.gson;

import com.google.gson.ExclusionStrategy;
import com.google.gson.FieldAttributes;

class Exclusion implements ExclusionStrategy {
    @Override
    public boolean shouldSkipField(FieldAttributes f) {
        return false;
    }

    @Override
    public boolean shouldSkipClass(Class<?> clazz) {
        return false;
    }
}

File: lib/src/main/kotlin/com/qwwuyu/lib/gson/Exclusion.kt
package com.qwwuyu.lib.gson

import com.google.gson.ExclusionStrategy
import com.google.gson.FieldAttributes

internal class Exclusion : ExclusionStrategy {
    override fun shouldSkipField(f: FieldAttributes): Boolean {
        return false
    }

    override fun shouldSkipClass(clazz: Class<*>?): Boolean {
        return false
    }
}